config/site_settings.yml

```diff
@@ -1326,7 +1310,7 @@ user_preferences:
   default_email_mailing_list_mode: false
   default_email_mailing_list_mode_frequency:
     enum: 'MailingListModeSiteSetting'
-    default: 0
+    default: 1
```

Comment:
In `MailingListModeSiteSetting`, only 1 and 2 are valid values. 0 is removed 2 months ago